src/app/auth/register/page.tsx

```diff
@@ -0,0 +1,121 @@
+                    className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                  />
+                </div>
+              </div>
+
+              <div>
+                <label className="block text-sm font-medium leading-6 text-white-900">
+                  Profile Picture
+                </label>
+                <div className="mt-2">
+                  <input
+                    name="profilePicture"
+                    type="url"
+                    autoComplete="profilePicture"
+                    placeholder="  Just do it , you are gorgeous 💗 "
+                    onChange={(e) => setProfilePicture(e.target.value)}
+                    className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                  />
+                </div>
+              </div>
+
+              <div>
+                <label className="block text-sm font-medium leading-6 text-white-900">
+                  Status Message
+                </label>
+                <div className="mt-2">
+                  <input
+                    name="statusMessage"
+                    type="text"
+                    autoComplete="statusMessage"
+                    placeholder="  We Know it is empty , please try ...🤯 "
+                    onChange={(e) => setStatusMessage(e.target.value)}
+                    className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                  />
+                </div>
+              </div>
+              <p className="mt-10 text-center text-sm text-gray-500">
+                No Name in Mind ?&nbsp;
+                <a
+                  href="https://namegenerator.com/people/full"
+                  target="_blank"
+                  className="font-semibold leading-6 text-indigo-600 hover:text-indigo-500"
+                >
+                  let us help
+                </a>
+              </p>
+              <div>
+                <button
+                  type="submit"
+                  className="flex w-full justify-center rounded-md bg-indigo-600 px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                  onClick={onClickRegister}
+                >
+                  Sign Up
+                </button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default Register;
```

Comment:
This file is a bit long, see if you can make it shorter(less than 100 lines) by creating subcomponents